docs/MISSION_AND_VALUES.md

```diff
@@ -0,0 +1,80 @@
+open, common, standard** -- all using either language agnostic or pure Python tooling.
+
+Looking to the existing microscopy community that have centralized around the
+[Open Microscopy Environment](https://www.openmicroscopy.org/) (OME) standard -- we also
+describe our mission as:
+
+> AICSImageIO provides a method to fully convert from an existing proprietary microscopy
+> file format to, or _emulate_, a Python representation of the OME model regardless of
+> image size, format, or location.
+
+We hope to accomplish this by:
+* being **easy to use and install**. We will take extra care to ensure that this library
+is easy to use and fully installable on Windows, Mac-OS, and Ubuntu.
+* being **well-documented** with our entire API having up-to-date, useful docstrings
+and additionally providing examples of more complex use-cases when possible.
+* providing a **consistent and stable API** to users by following
+[semantic versioning](https://semver.org/) and limiting the amount of breaking changes
+introduced unless necessary for the future robustness or scalability of the library.
+* sustaining **comparable or better performance when compared to more tailored file
+format reading libraries**. We will regularly run benchmarks utilizing a set of varied
+size images from all the file formats the library is capable of reading.
+* **working closely with the microscopy community** while deciding on standards and best
+practices for open, accessible, file formats and imaging and in deciding which
+proprietary file formats and metadata selection are in need of support.
+
+## Values
+* We are **inclusive**. We welcome and mentor newcomers who are making their first
+contribution and strive to grow our most dedicated contributors into core developers. We
+have a [Code of Conduct](./CODE_OF_CONDUCT.html) to ensure that the AICSImageIO remains
+a welcoming place for all.
+* We are **community-driven**. We respond to feature requests and proposals on our
+[issue tracker](https://github.com/AllenCellModeling/aicsimageio/issues) and make
+decisions that are driven by our user's requirements.
+* We focus on **file IO and metadata conversion**, leaving image analysis functionality
+and visualization to other libraries.
+* We aim to **develop new methods of metadata extraction and conversion**, instead of
+duplicating existing, or porting, from other libraries primarily by creating **language
+agnostic methods** for metadata manipulation.
+* We value **simple, readable implementations**. Readable code that is easy to
+understand, for newcomers and maintainers alike, makes it easier to contribute new code
+as well as prevent bugs.
+* We value **education and documentation**. All functions should have docstrings,
+preferably with examples, and major functionality should be explained in our tutorials.
+Core developers can take an active role in finishing documentation examples.
+* We **minimize [magic](https://en.wikipedia.org/wiki/Magic_(programming))** and always
```

Comment:
I dig this point!

Reply:
A lot of this was borrowed from the napari dev docs so credit to them there. But **totally** agree on this one and it's something I think we have failed at previously with the mixed dask routing -- something I hope is resolved in the 4.0 API :)